data_collector.py

```diff
@@ -14,34 +14,34 @@
 
 def record_video(fps, seconds, scale_factor=1, meta_data={}):
     number_of_frames = int(fps * seconds)
-    
+
     # write something that makes the video go to server
     cap = cv2.VideoCapture(0)
-    
+
     local_name = 'basicvideo.mp4'
     width = int(cap.get(cv2.CAP_PROP_FRAME_WIDTH)*scale_factor) # 640
     height = int(cap.get(cv2.CAP_PROP_FRAME_HEIGHT)*scale_factor) # 480
     writer = cv2.VideoWriter(local_name, cv2.VideoWriter_fourcc(*'MP4V'), fps, (width, height))
-    
+
     counter = 0
     while True:
         counter += 1
-        
+
         ret, frame = cap.read()
         frame = cv2.resize(frame, (0,0), fx=scale_factor, fy=scale_factor)
         writer.write(frame)
-    
+
         cv2.imshow('frame', frame)
-    
-        if (cv2.waitKey(1) & 0xFF == ord('q')) or counter == number_of_frames: 
+
+        if (cv2.waitKey(1) & 0xFF == ord('q')) or counter == number_of_frames:
             break
-        
-    
+
+
     cap.release()
     writer.release()
     cv2.destroyAllWindows()
     dt = datetime.now().strftime('%Y-%m-%d %H:%M:%S')
-    remote_name = r'/home/srd290_lab1/Documents/RockPaperScissors/basic'+dt+'.mp4'
+    remote_name = f'/home/srd290_lab1/Documents/RockPaperScissors/data/'+meta_data['Hand']+'_'+meta_data['Name']+'_'+dt+'.mp4'
```

Comment:
The 'r' is for the computer to read it as a raw string, which helps in viewing it as a path. There is no need in using 'f' as we don't use it's functionality